### PR TITLE
Support series API in logcli

### LINKED ISSUE #1841: Support series API in logcli
**Describe the solution you'd like**

Since there is already a series API to get the different series, so it would be nice to add this functionality in logcli.

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #1841 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Comment by cyriltovena** [comment]
If you have some time to rebase and fix the conflict, I'll get that merged asap.

**Comment by yeya24** [comment]
Hi @cyriltovena  @owen-d  I have rebased my code. PTAL

File: cmd/logcli/main.go [hunk 1]
```diff
@@ -165,49 +170,76 @@ func newLabelQuery(labelName string, quiet bool) *labelquery.LabelQuery {
...
+		q.Start = mustParse(from, defaultStart)
+		q.End = mustParse(to, defaultEnd)
+		q.Quiet = *quiet
+		return nil
+	})
+
+	cmd.Flag("since", "Lookback window.").Default("1h").DurationVar(&since)
+	cmd.Flag("from", "Start looking for logs at this absolute time (inclusive)").StringVar(&from)
+	cmd.Flag("to", "Stop looking for logs at this absolute time (exclusive)").StringVar(&to)
+	cmd.Flag("match", "eg '{foo=\"bar\",baz=~\".*blip\"}'").Required().StringsVar(&q.Matchers)
```
Comment (cyriltovena) [inline cmd/logcli/main.go]: Can you show me an example of how it work with multiple matches ? what's the cli look like for them ? is it space based or comma seperated ?

Comment (cyriltovena) [reply cmd/logcli/main.go]: I'm asking cause I see that `StringsVar` is an array.

Comment (owen-d) [reply cmd/logcli/main.go]: I'd also like to know that, but otherwise LGTM!

Nice work @yeya24

Comment (yeya24) [reply cmd/logcli/main.go]: image
This works just the same like `promtool query series`

Comment (owen-d) [reply cmd/logcli/main.go]: Great. I won't hold up the PR if you don't want to add the docs, but I'd appreciate it :). Let me know.

Comment (yeya24) [reply cmd/logcli/main.go]: Will add it this afternoon. Will ping you when I finish

Comment (yeya24) [reply cmd/logcli/main.go]: Hey, I have updated the docs. PTAL @owen-d

**Pull request merged**